code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -4449,6 +4449,30 @@
 	reagents.add_reagent(NUTRIMENT, 3)
 	reagents.add_reagent(SUGAR, 2)
 	icon_state = "sweet[rand(1,12)]"
+	if (icon_state == "sweet1")
```

Comment:
It might be more elegant to instead do something like

`var/list/flavors = list("Strawberry","Lime","Blueberry",etc)
var/variety = rand(1,12)
icon_state = "sweet[variety]
name = "[flavors[variety]] sweet"`

Reply:
Yeah I already mentioned this on discord. I was being lazy and I should really fix this. I just wanted to get something working first.